h2gis-functions/pom.xml

```diff
@@ -108,6 +108,11 @@
             <version>${project.parent.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
slf4j-simple has been removed by @SPalominos

Reply:
It should have not been removed but placed under the test scope. Because logger is silent in unit test without it.